app/src/main/res/values/strings.xml

```diff
@@ -193,6 +194,8 @@
     <string name="app_settings_distrust_system_certs">Distrust system certificates</string>
     <string name="app_settings_distrust_system_certs_on">System and user-added CAs won\'t be trusted</string>
     <string name="app_settings_distrust_system_certs_off">System and user-added CAs will be trusted (recommended)</string>
+    <string name="app_settings_distrust_system_certs_dialog_title">Attention!</string>
+    <string name="app_settings_distrust_system_certs_dialog_message">Enabling this option will cause DAVx5 to distrust system certificates. You will need to manually install the required certificates. This option is intended for advanced users.</string>
```

Comment:
We shouldn't use DAVx5 in strings, because the app name can change. If we use it, we should use %s and insert the app name in the code.

However I suggest to not use the wording "enable / disable" because people may not know whether "enable" means "enable the option: distrust system certificates" or "enable system certificates".

So my suggestion:

> [Icon] Attention sign
> [Title] Distrust system certificates
>
> If this setting is active, system certificates are not considered as trustworthy. This means that you will have to manually accept every certificate (also when the server renews its certificate).
>
> [Cancel] / [Distrust]

Reply:
Sounds good, but it lacks what the people will expect: that the connection will not work... The message is not threationing enough. This has to be added somehow in the text since many users enable it accidentially or just because to try something and if we don't write that connection will be not working probably they will still distrust, not understanding that this option can prevent them from syncing or creating the account

Reply:
Well it shouldn't? They should still be able to accept every certificate manually and create an account / sync?

Reply:
@ArnyminerZ I've now extensively talked regarding this with @rfc2822 and even to me this option was not fully clear as it seems `:) So lets go with this text: 

[Icon] Attention sign
[Title] Distrust system certificates

"If this setting is active, system certificates are not considered as trustworthy. This means that you will have to manually accept every certificate (also when the server renews its certificate) or synchronization will not work."